### PR TITLE
Implemented Contact Us page

### PR DESCRIPTION
**Pull request by zans-laksa**

### CODE REVIEW

File: src/app/component/ContactForm/ContactForm.container.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        sendMessage(fields)
+            .then(
+                /** @namespace Component/ContactForm/Container/sendMessageThen */
+                () => this.setState({ isLoading: false })
+            )
+            .catch(
+                /** @namespace Component/ContactForm/Container/sendMessageThenCatch */
+                () => this.setState({ isLoading: false })
+            );
```
Comment (yeegor) [inline src/app/component/ContactForm/ContactForm.container.js:60]: I believe this error is unhandled, is it? On message send fail user will not be notified about it. Please provide a notification!

File: src/app/query/ContactForm.query.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+import { Field } from 'Util/Query';
+
+/** @namespace Query/ContactForm */
+export class ContactFormQuery {
+    getSendContactFormMutation(options) {
+        return new Field('contactForm')
+            .addArgument('contact', 'ContactForm!', options)
+            .addField('message');
```
Comment (yeegor) [inline src/app/query/ContactForm.query.js:19]: Please split this into separate method in order to make it more easily overridable.
With the current implementation, if custom theme's developer adds a custom field to this mutation (and to the query above - same situation), he'd need to override the query (mutation) completely, hence if afterwards we'll want to add something from our side - he will not receive this update. See the other queries in the application for a reference.

**Review by yeegor — CHANGES_REQUESTED** [review]
Overall code quality is very high!
My comments are related to very scandipwa-specific details, and there is not a lot of them.
Nice job!

File: src/app/component/ContactForm/ContactForm.container.js (hunk 1 above)
Comment (zans-laksa) [reply src/app/component/ContactForm/ContactForm.container.js]: Actually this is handled in a dispatcher.
There were some copy paste code.
Refactored it a bit.

File: src/app/query/ContactForm.query.js (hunk 2 above)
Comment (zans-laksa) [reply src/app/query/ContactForm.query.js]: Done.

**Pull request merged**